.circleci/config.yml

```diff
@@ -177,7 +177,6 @@ jobs:
       - run:
           name: Protoc version
           command: |
-            # tail 2 + head 1 because CircleCI adds an extra line
```

Comment:
Isn't it good to have this explanation for the weird tail|head below?

Reply:
put back.